airbyte-server/src/main/java/io/airbyte/server/converters/CatalogConverter.java

```diff
@@ -34,9 +34,34 @@
  */
 public class CatalogConverter {
 
+  private static io.airbyte.api.model.AirbyteStreamName toApi(final String name, final io.airbyte.protocol.models.AirbyteStreamName streamName) {
```

Comment:
streamname should be the first parameter since it is the one that will remain. 

Can you add a Preconditions that checks that only one of the two is set? (assuming this is correct)

Reply:
for the moment both are set, sometimes the StreamName object could be null (if it was serialized before introducing the namespace field)